src/app/advancedsearch/advancedsearch.component.spec.ts

```diff
@@ -22,7 +22,7 @@ describe('AdvancedsearchComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
+  it('should create an instance', () => {
     expect(component).toBeTruthy();
   });
-});
+});
```

Comment:
Always try to leave a extra line at the end of the file.
Do it for every other file.